src/panels/config/automation/condition/types/ha-automation-condition-sun.ts

```diff
@@ -0,0 +1,107 @@
+import "@polymer/paper-input/paper-input";
+import "@polymer/paper-radio-button/paper-radio-button";
+import "@polymer/paper-radio-group/paper-radio-group";
+// tslint:disable-next-line
+import { PaperRadioGroupElement } from "@polymer/paper-radio-group/paper-radio-group";
+import { LitElement, customElement, property, html } from "lit-element";
+import { HomeAssistant } from "../../../../../types";
+import {
+  SunCondition,
+  handleChangeEvent,
+  ConditionElement,
+} from "../ha-automation-condition-row";
+import { fireEvent } from "../../../../../common/dom/fire_event";
+
+@customElement("ha-automation-condition-sun")
+export class HaSunCondition extends LitElement implements ConditionElement {
+  @property() public hass!: HomeAssistant;
+  @property() public condition!: SunCondition;
+
+  public static get defaultConfig() {
+    return {};
```

Comment:
Shouldn't this include sunrise / sunset  ?

Reply:
No, you can't remove them when set, and since you could only set before and not set after or otherwise, a default doesn't make sense?